modules/bringauto_git/Git.go

```diff
@@ -53,6 +57,21 @@ func (args *GitCheckout) ConstructCMDLine() []string {
 	}
 }
 
+func (args *GitGetHash) ConstructCMDLine() []string {
+	validateGITPath(args.ClonePath)
+	cmd := []string{
+		GitExecutablePath,
+		"log",
+		"--pretty=format:'%H'",
```

Comment:
Maybe use a different pretty command and unmarshal json?
`git log --pretty=format:'{"commit":"%H"}`?